### DISCUSSION TITLE
Timer countdown

### DISCUSSION THREAD

**Discussion by Mauro2022mauro** [sircharlo/meeting-media-manager#1153] [Ideas] [answered]
This program is perfect thank you for your valuable work. Do you think it's possible to add a countdown timer for the speakers?

**Comment by sircharlo** [comment]
> This program is perfect thank you for your valuable work. Do you think it's possible to add a countdown timer for the speakers?

@Mauro2022mauro, thanks! What do you mean exactly, a countdown timer for the speakers? Where would the timer be displayed? What would be its purpose?

**Comment by Mauro2022mauro** [comment]
I am writing from Italy and sorry for my English.
In our congregation we have a PC that manages the meetings for those connected with Zoom, JW library and a timer program for those who have a part on the podium to see that the allotted time does not go out. This appears on a small screen.
We are looking for a program to simplify, since if we activate all three software together they conflict with each other.
Back then I was thinking of just using your Meeting Media Manager software and Zoom. Eliminating JW library and our countdown software.
I hope I have clarified the reason for a timer.
A greeting and thanks again.

**Reply by mtdvlpr** [reply]
@Mauro2022mauro, where would you like to see this timer? And how would it work? Do you want to set a time and start/stop it yourself and set another timer again?

**Comment by Mauro2022mauro** [comment]
Yes in a second monitor screen, and set time (countdown) and start and stop.

Il giorno mer 11 gen 2023 alle ore 18:50 Manoah Tervoort <
***@***.***> ha scritto:

**Comment by Mauro2022mauro** [comment]
an example https://timerliveofficial.wordpress.com/it/
Thanks

Il giorno mer 11 gen 2023 alle ore 18:53 Mauro Aliani <
***@***.***> ha scritto:

**Comment by Chipperchap** [comment]
For what it is worth in the four congregations I assist we use two computers. We found we were getting too many screen conflicts, especially after each windows update. 

- One PC cares for all media and host of ZOOM. 

- The other PC cares for ZOOM co-host audio (home participant 'raise hand' comments) and the timer screen – a small “foldback” screen located above our A/V booth for the stage to view.  (Need LDC approval to install).

We use OBS ‘virtual camera’ to display the timer also for the ZOOM participants as it has a meeting start countdown timer as well. 
We just find this the easiest as the host PC already controls the booth screen and the auditorium screens, so adding another screen and controlling the timer was yet another task for the brother to care for, so we share the load. 

I cannot comment on the program referenced above as I have not seen it before, but we find [OnlyT](https://github.com/AntonyCorbett/OnlyT) (by Antony Corbett – AKA the author of “SoundBox”) to be intuitive and it already “knows” the timing of each section of the meeting when we arrive at the K-Hall each week.

![Screenshot 2023-01-12 at 8 13 56 AM](https://user-images.githubusercontent.com/35364657/211897916-8c325351-c102-4a3b-86bf-eb3c0d71f0a3.png)

**Comment by Mauro2022mauro** [comment]
Thank you so much for your valuable suggestions.

Mauro

**Comment by mtdvlpr** [comment]
@Mauro2022mauro, looking at both examples provided here, it seems like a whole other application that would need to be recreated from scratch to be included in M3. This would add a lot of extra complexity and feels like redundant work since such timer applications already exist. In my opinion, this is not something we should implement ourselves. 

However if you know of a way we could integrate the existing applications (like we integrate OBS) without recreating the whole thing I would definitely be willing to look into that.

@sircharlo, what are your thoughts on this matter?

**Comment by Mauro2022mauro** [comment]
Thank you for your suggestion.
As you already told me, let's see how to use the software you recommended Only T. Thank you.

**Comment by sircharlo** [comment, marked as answer]
Yeah, definitely outside of the scope of M3. I don't think there's much to do here, especially considering the excellent OnlyT software that is available.
